EIPS/eip-2384.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 2384
+title: Istanbul/Berlin Difficulty Bomb Delay
+author: Eric Conner (@econoar)
+discussions-to: https://gitter.im/ethereum/AllCoreDevs?at=5dd4ce8050010612b2eeb9d6
```

Comment:
I think a github issue or a FEM forum is a better place. Gitter gets a lot of traffic from other discussions too.

Reply:
There is nowhere else ATM

Reply:
We have no idea how long Gitter allows access to the history, while we know that FEM/Github has worked well for every other EIP.

Why are you against having a proper forum for it?